### ISSUE TITLE
I would like to ask something.

### ISSUE THREAD

**Issue by benzkaka** [xAranaktu/FC-25-CM-Cheat-Table#3]
https://github.com/xAranaktu/FC-24-CM-Cheat-Table/issues/31

Has this issue been resolved yet in FC25 ?

**Comment by xAranaktu** [comment]
No. As you can read there futbin has been updated and it won't be fixed.
